db/schema.rb

```diff
@@ -77,6 +77,7 @@
 
   create_table "financial_transactions", force: true do |t|
     t.integer  "ordergroup_id",                         default: 0, null: false
+    t.integer  "cost_type,                              default: 0, null: false
```

Comment:
Hey, you're missing a [migration](http://edgeguides.rubyonrails.org/active_record_migrations.html) here :o